cmd/earthly/flags.go

```diff
@@ -319,10 +319,18 @@ func (app *earthlyApp) buildFlags() []cli.Flag {
 		&cli.StringFlag{
 			Name:        "org",
 			EnvVars:     []string{"EARTHLY_ORG"},
-			Usage:       wrap("The name of the organization the satellite belongs to. ", "Required when using --satellite and user is a member of multiple organizations."),
+			Usage:       wrap("The name of the organization that the satellite belongs to. ", "Required when using --satellite and user is a member of multiple organizations."),
```

Comment:
should we set this to `Hidden: true,` to match the `project` flag?

I think this was overlooked in a previous PR.

Reply:
I'm not sure, but I'm happy to update it if that's preferred.

Reply:
up to you... just something that jumped out.